app/src/main/java/com/stevesoltys/seedvault/settings/SettingsViewModel.kt

```diff
@@ -233,4 +237,27 @@ internal class SettingsViewModel(
         BackupJobService.cancelJob(app)
     }
 
+    fun onLogcatUriReceived(uri: Uri?) = viewModelScope.launch(Dispatchers.IO) {
+        if (uri == null) {
+            onLogcatError()
+            return@launch
+        }
+        val command = "logcat -d --uid=1000,${myUid()} *:V"
```

Comment:
maybe add a comment here
// 1000 is system uid, needed to get backup logs from the OS code.